### PR TITLE
Unit tests for the new labels module

### PR DESCRIPTION
**Pull request by frheault**
Following PR #648 merged from @EmmaRenauld, these are simple unit tests to make sure the expected behavior stays the same. There is little to no edge case, no tentative to make it crash or anything.

Later if desired, these could be extended to be more robust, which could be internships for bachelor students or "get to know our code and git" activities for new students.

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Review by EmmaRenauld — COMMENTED** [review]
Good. Everything runs locally (12 tests in 0.16 sec). A few small comments.

File: scilpy/image/tests/test_labels.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+ref_out_labels = deepcopy(ref_in_labels)
+for i in range(1, 8, 2):
+    ref_out_labels[ref_out_labels == i] = 0
+
+
+def test_combine_labels_all_labels():
+    in_labels = deepcopy(ref_in_labels)
+    out_labels = combine_labels([in_labels], [np.unique(ref_in_labels)],
+                                ('all_labels',), background_id=0,
```
[the remark this answers is not in the document]
Comment (frheault) [reply scilpy/image/tests/test_labels.py:31]: This is tested in `test_combine_labels_group_in_m`

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Review by EmmaRenauld — APPROVED** [review]
